.env.dist

```
@@ -5,6 +5,7 @@
 VELOCITA_URL=http://localhost:8080
 VELOCITA_HTTP_PORT=8080
 VELOCITA_HTTPS_PORT=8443
+VELOCITA_IPV6_ENABLED=true
```

```suggestion
#VELOCITA_IPV6_LISTEN=true
```

1. Let's name this "listen" instead of "enabled" since it does not affect IPv6 connectivity as a whole, but just on which interfaces nginx is listening.
2. The default value should be `true`, so this is an optional setting.

---

check